deploy/k8s/charts/trustification/templates/helpers/_guac.tpl

```diff
@@ -19,12 +19,15 @@ Arguments (dict):
   * module - module object
 */}}
 {{- define "trustification.guac.graphql.envVars"}}
-{{- if eq ( include "trustification.openshift.useServiceCa" .root ) "true" }}
+{{- if and
```

Comment:
The GraphQL endpoint TLS configuration shouldn't be influenced by the TLS configuration between guac and the database. Those are two different things.

If you want to disable the TLS endpoint of GUAC graphql, you can change this through the settings of the "useServiceCa" template.

Reply:
For OpenShift I need `useServiceCa` for everything else but this part. Howe could we have it off only for GUAC, and only when the database is not configured to support TLS?

Reply:
There should be no connection between the database and the graphql endpoint.

If we really need such an option, then let's plan this for post tech-preview and add this as a proper option.

Reply:
> I think there's a relationship:

I am sorry, I don't see it. There are two independent options: 1) TLS settings for the Postgres database 2) TLS settings for the GUAC GraphQL endpoint. … We must not create an artificial connection between them.